patches/tModLoader/Terraria/WorldGen.cs.patch

```diff
@@ -2690,6 +2883,17 @@
  		return false;
  	}
  
+@@ -50401,7 +_,9 @@
+ 		if (!Main.remixWorld)
+ 			return;
+ 
++		if (TileLoader.Convert(x, y, 0)) {
```

Comment:
Whole method is an `else if` chain so you could convert this into a guard statement for cleaner code and to avoid patching the next line